sustc-api/src/main/java/io/sustc/dto/UserRecord.java

```diff
@@ -1,44 +1,70 @@
 package io.sustc.dto;
 
-import com.opencsv.bean.CsvBindByName;
 import lombok.Data;
 
 @Data
 public class UserRecord {
 
-    @CsvBindByName(column = "Mid")
-    private Long mid;
+    /**
+     * The user's ID
+     */
+    private long mid;
 
-    @CsvBindByName(column = "Name")
+    /**
+     * The user's name
+     */
     private String name;
 
-    @CsvBindByName(column = "Sex")
+    /**
+     * The user's sex
+     */
     private String sex;
 
-    @CsvBindByName(column = "Birthday")
+    /**
+     * The user's birthday
+     */
     private String birthday;
 
-    @CsvBindByName(column = "Level")
-    private Short level;
+    /**
+     * The user's level
+     */
+    private int level;
 
-    @CsvBindByName(column = "Sign")
-    private String sign;
+    /**
+     * The user's current number of coins
+     */
+    private int coin;
 
-    @CsvBindByName(column = "following")
-    private Long[] following;
+    /**
+     * The user's personal sign, can be empty
```

Comment:
meaning that the value would never be `null`, but `""`?

Reply:
ok